public/js/angular/controllers.js

```diff
@@ -1,6 +1,6 @@
 angular
-  .module('webmakerApp')
-  .controller('navigationController', ['$scope', '$location', '$routeParams', '$rootScope', 'weblit', 'wmNav', 'CONFIG',
+  .module("webmakerApp")
```

Comment:
in this case, given the size of the refactor, it's pretty much the only time we can safely go "let's pick one or the other", so I'd say let's go for it: single quotes in all the files